packages/core/scripts/build.mjs

```diff
@@ -0,0 +1,57 @@
+import esbuild from "esbuild";
+import path from "node:path";
+import fs from "node:fs";
+import { deleteSync } from "del";
+import { fileURLToPath } from "node:url";
+import glob from "fast-glob";
+
+const args = process.argv.slice(2);
+const createEntryFile = args[0] === "--create-entry";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const buildFolder = path.join(__dirname, "../../../dist/salt-ds-core/css");
+const absWorkingDir = path.resolve(__dirname, "..");
+const entryFile = "src/salt-core.css";
+
+/** MAIN */
+deleteSync([buildFolder], { force: true });
+createCssEntryFile(createEntryFile);
+runBuild();
+
+function createCssEntryFile(shouldCreate) {
+  if (shouldCreate) {
```

Comment:
Should this always run as part of build process? Otherwise it can be easily forgotten when new components are added

Reply:
If we can add a new build step in do it first then sure...the issue I had was that 3 build tasks get kicked off in parrallel all deleting and recreating the same file so the build failed a lot

Reply:
Fixed